src/frontends/tensorflow/src/op/unpack.cpp

```diff
@@ -19,9 +19,16 @@ OutputVector translate_unpack_op(const NodeContext& node) {
     auto num = node.get_attribute<int64_t>("num");
 
     auto axis_const = make_shared<Constant>(element::i64, Shape{}, axis);
-    auto res = make_shared<Split>(input, axis_const, num);
-    set_node_name(node.get_name(), res);
-    return res->outputs();
+    auto split = make_shared<Split>(input, axis_const, num);
+    set_node_name(node.get_name(), split);
+    OutputVector res;
+    int i = 0;
+    for (auto out : split->outputs()) {
+        auto squeezed_res = make_shared<Squeeze>(out, axis_const);
+        set_node_name(node.get_name() + "/squeeze_" + to_string(i++), squeezed_res);
```

Comment:
we should save original node name, without "/squeeze_"

Reply:
We cannot save friendly_name, but I agree we should save tensor names